ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/Config.scala

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http.perf
+
+import com.daml.jwt.JwtDecoder
+import com.daml.jwt.domain.Jwt
+import com.daml.ports.Port
+import scopt.RenderingMode
+
+private[perf] final case class Config(
+    jsonApiHost: String,
+    jsonApiPort: Port,
+    scenario: String,
+    jwt: Jwt,
+    packageId: Option[String],
+) {
+  override def toString: String =
+    s"Config(jsonApiHost=${this.jsonApiHost}, " +
+      s"jsonApiPort=${this.jsonApiPort.value: Int}, " +
+      s"scenario=${this.scenario}, " +
+      s"jwt=..., " + // don't print the JWT
+      s"packageId=${this.packageId.toString})"
+}
+
+object Config {
```

Comment:
```suggestion
private[perf] object Config {
```

Then the members don't need `private[perf]`.